packages/libs/codemodel/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@autorest/codemodel",
+  "version": "4.14.0",
+  "patchOffset": -1219,
```

Comment:
What does the negative do in this case?

Reply:
Makes it that the next version is 4.14.0 instead of being 4.14.1220. I can revert if we don't care but it was quite a jump